Assets/Scripts/UI/UITrickDetetctor.cs

```diff
@@ -10,7 +10,7 @@
 public class UiScoreSystem : MonoBehaviour, TrickObserver
 {
     [SerializeField] private PlayerSubject _player;
-    [SerializeField] private List<Collectible> _collectibes;
+    [SerializeField] private GameObject _collectibesParent;
```

Comment:
Inconsistent use of the "_" prefix convention for private variables, this practice isn't adopted elsewhere in the codebase. On another note we use both "public" and "[SerializeField] private" to denote variables visible to the editor, I suggest we converge on the latter going forward.